### PR TITLE
Ds 27 style top news desktop

### LINKED ISSUE #27: Finish desktop style on TodayNews section
**Is your feature request related to a problem? Please describe.**
TodayNews section looks like crap with current styling. This is not acceptable.

### PR DESCRIPTION
**Pull request by dannysanchez559**
## Changes
1. Style entire app for desktop version
2. Work on grouping for news sections and adjust article pattern

## Purpose
To make app look closer to figma proto.

Closes #27

### CODE REVIEW

File: modern-news-app/public/index.html [hunk 1]
```diff
@@ -24,6 +24,10 @@
       work correctly both with client-side routing and a non-root public URL.
       Learn how to configure a non-root public URL by running `npm run build`.
     -->
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Public+Sans:wght@600&display=swap" rel="stylesheet"> 
+
```
Comment (krferrell) [inline modern-news-app/public/index.html:30]: ooo Very nice

**Review by JayDampitan — APPROVED** [review]
Looks good Dan!

**Review by BlackCubes — APPROVED** [review]
LGTM!

**Review by krferrell — APPROVED** [review]
Looks great!

**Review by Gjhnsn — APPROVED** [review]
Good work Danny, excited to see it!

**Pull request merged**